solc/CommandLineInterface.cpp

```diff
@@ -1379,6 +1403,18 @@ bool CommandLineInterface::processInput()
 		}
 	}
 
+	if (m_args.count(g_argModelCheckerEngine))
+	{
+		string engineStr = m_args[g_argModelCheckerEngine].as<string>();
+		optional<ModelCheckerEngine> engine = ModelCheckerEngine::fromString(engineStr);
```

Comment:
@cameel This is what I mean. If I use the variables here this block will actually get larger.

Reply:
Oh, thanks for the pointer. I was actually looking at the old version.

In that case obviously I wouldn't use them here.

For `value_name()` it would be best if you just moved the variables to SMT checker and used the same constants in both it and in `fromString()`... but to be honest it's additional hassle for a marginal gain so at this point I'd just leave this as it is now.

Reply:
Or, if you want to remove them, I'm fine with that too. For me the biggest benefit is in avoiding accidental typos. With how short and few they are, that's not a big risk here.

Reply:
Yea, I think it's overkill to add variables for them.